common/tools/dev-tool/src/commands/run/build-package.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { concurrently, ConcurrentlyCommandInput } from "concurrently";
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+import { createPrinter } from "../../util/printer";
+import { run } from "../../util/run";
+
+const log = createPrinter("build-package");
+
+export const commandInfo = makeCommandInfo("build-package", "build a package for production");
+
+export default leafCommand(commandInfo, async () => {
+  log.info("Building package with tshy...");
```

Comment:
Some debugging code here, I'll resolve this comment once it's ready for another review